packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -685,7 +685,7 @@ export const getNodesWithSettings = ( tree, blockSelectors ) => {
 	// Top-level.
 	const presets = pickPresets( tree.settings );
 	const custom = tree.settings?.custom;
-	if ( ! isEmpty( presets ) || custom ) {
+	if ( Object.keys( presets ).length > 0 || custom ) {
```

Comment:
`presets` will always be an object:

https://github.com/WordPress/gutenberg/blob/4a6972b681196432663b9f4c5ce53efb8ba80b49/packages/block-editor/src/components/global-styles/use-global-styles-output.js#L675